pkgs/servers/nextcloud/patches/v26/0001-Setup-remove-custom-dbuser-creation-behavior.patch

```diff
@@ -0,0 +1,157 @@
+From 2e32658e52e196ef2e3ef671606da9131f0079d9 Mon Sep 17 00:00:00 2001
+From: Raito Bezarius <masterancpp@gmail.com>
+Date: Tue, 21 Mar 2023 16:38:29 +0100
+Subject: [PATCH] Setup: remove custom dbuser creation behavior
+
+Both PostgreSQL and MySQL can be authenticated against from Nextcloud by
+supplying a database password. Now, during setup the following things
+happen:
+
+* When using postgres and the db user has elevated permissions, a new
+  unprivileged db user is created and the settings `dbuser`/`dbpass` are
+  altered in `config.php`.
+
+* When using MySQL, the password is **always** regenerated since
+  24.0.5/23.0.9[1].
+
+I consider both cases problematic: the reason why people do configuration
+management is to have it as single source of truth! So, IMHO any
+application that silently alters config and thus causes deployed
+nodes to diverge from the configuration is harmful for that.
+
+I guess it was sheer luck that it worked for so long in NixOS because
+nobody has apparently used password authentication with a privileged
+user to operate Nextcloud (which is a good thing in fact).
+
+[1] https://github.com/nextcloud/server/pull/33513
+
+Original-Author: Maximilian Bosch <maximilian@mbosch.me>
+---
+ lib/private/Setup/MySQL.php      | 61 +++-----------------------------
+ lib/private/Setup/PostgreSQL.php | 39 --------------------
+ 2 files changed, 4 insertions(+), 96 deletions(-)
+
+diff --git a/lib/private/Setup/MySQL.php b/lib/private/Setup/MySQL.php
+index 50f566728a9..ce721415e30 100644
+--- a/lib/private/Setup/MySQL.php
++++ b/lib/private/Setup/MySQL.php
+@@ -145,62 +145,9 @@ class MySQL extends AbstractDatabase {
+ 	 * @param IDBConnection $connection
+ 	 */
+ 	private function createSpecificUser($username, $connection): void {
+-		$rootUser = $this->dbUser;
+-		$rootPassword = $this->dbPassword;
```

Comment:
so we can drop https://github.com/NixOS/nixpkgs/blob/nextcloud26/pkgs/servers/nextcloud/patches/v25/0001-Setup-remove-custom-dbuser-creation-behavior.patch#L37-L38 , too?

Reply:
as a cleanup PR for v25 yes, but I wouldn't bother too much? you can do it :P